spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,6 @@
+def sign_in(user)
+  visit new_user_session_path
+  fill_in "Email", with: user.email
+  fill_in "Password", with: user.password
+  click_button "Log in"
+end
```

Comment:
I think this is an awesome method to have defined, it'll be super useful with your future tests! Nice work!